azext_iot/dps/_help.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for IoT Hub commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_deviceprovisioningservice_help():
+    helps["iot device registration"] = """
+        type: group
+        short-summary: Manage IoT Device registrations through the IoT Device Provisioning Service.
+        long-summary: Use `az iot dps enrollment registration` or `az iot dps enrollment-group registration` to view and delete registrations.
+    """
+
+    helps["iot device registration create"] = """
+        type: command
+        short-summary: Register an IoT Device with the IoT Device Provisioning Service.
```

Comment:
The 'Device' in IoT Device is not a proper noun and should be lowercase d. There are a lot of occurrences.